test/integration/suites/node-attestation/conf/server.conf

```diff
@@ -0,0 +1,30 @@
+server {
+	bind_address = "0.0.0.0"
+	bind_port = "8081"
+	registration_uds_path = "/tmp/spire-registration.sock"
+	trust_domain = "domain.test"
+	data_dir = "/opt/spire/data/server"
+	log_level = "DEBUG"
+	ca_ttl = "1h"
+	default_svid_ttl = "10m"
+	experimental {
+		enable_api = true
+	}
+}
+
+plugins {
+	DataStore "sql" {
+		plugin_data {
+			database_type = "sqlite3"
+			connection_string = "/opt/spire/data/server/datastore.sqlite3"
+		}
+	}
+	NodeAttestor "x509pop" {
```

Comment:
mmm why join token works without

```
    NodeAttestor "join_token" {
        plugin_data {
        }
    }
```
may it fails? @amartinezfayo

Reply:
I don't think this is needed with the new implementation (Join Token attestation isn't implemented in a plugin, it's right in the API handler)

Maybe we should require it? But it is not required right now